README.md

```diff
@@ -107,6 +107,70 @@ end
 
 _Note: `drop_split` (a.k.a. `finish_split`) does not physically delete split data from mixpanel or Test Track's database._
 
+### Generating Split Migrations
+
+Split configuration changes can be generated using Rails generators that are included with the TestTrack Rails client.
+
+```
+rails generate test_track:migration add_name_of_split
+```
+
+will generate a timestamped migration file with the content
+
+```ruby
+class AddNameOfSplit < ActiveRecord::Migration
+  def change
+    TestTrack.update_config do |c|
+      c.split :name_of_split, control: 100, treatment: 0
+    end
+  end
+end
+```
+
+Different types of splits can be generated using the `--type/-t` flag with the options of `experiment`, `gate` (a feature flag), or `drop` (ending a split)
```

Comment:
Based on the comment above, if we use the name of the split to infer its meaning, then perhaps it makes more sense here to have a name that ends in `_enabled` or `Enabled` create a feature flag split, since the only difference there would be in the names of the variants.